.travis.yml

```diff
@@ -5,19 +5,20 @@ python:
     - "3.4"
     - "3.5"
     - "3.6"
+    - "3.7"
+    - "3.8"
 
 env:
-    - DJANGO=1.8.13
-    - DJANGO=1.9.6
-    - DJANGO=1.10.8
-    - DJANGO=1.11.16
-    - DJANGO=2.0.9
-    - DJANGO=2.1.2
+    - DJANGO=1.11.27
+    - DJANGO=2.0.13
+    - DJANGO=2.1.15
+    - DJANGO=2.2.9
+    - DJANGO=3.0.2
 
 install:
     - pip install Django==$DJANGO
-    - pip install coverage==4.5.1
-    - pip install python-coveralls==2.9.1
+    # PyYAML pinned to work on Python 3.4
```

Comment:
Dropping Python 3.4 is probably OK at this point no? 

(2.7 too TBH but...)

Reply:
Yeah I guess I was just being lazy. I think we can release this one as-is then do another python deprecation release.